guides/source/5_1_release_notes.md

```diff
@@ -292,7 +292,7 @@ Please refer to the [Changelog][action-mailer] for detailed changes.
 *   Allowed passing lambdas as values to the `default` method.
     ([Commit](https://github.com/rails/rails/commit/1cec84ad2ddd843484ed40b1eb7492063ce71baf))
 
-*   Added support for parameterized invocation of mailers to share before filters and defaults
+*   Added support for parameterized invocation of mailers to share before callbacks and defaults
```

Comment:
Changed release note to be consistent with overall language as well as the before_action in the PR itself, even though David wrote it :)